ui/src/router/components/form/components/router_config/RulesPanel.js

```diff
@@ -34,7 +28,7 @@ export const RulesPanel = ({
   };
 
   const addRuleButton = (
-    <EuiButton fullWidth onClick={onAddRule} isDisabled={routes.length < 2}>
+    <EuiButton fullWidth onClick={onAddRule} isDisabled={!routes.length}>
```

Comment:
Tiny nit: since `routes.length == 0` can never happen (due to the changes you've made [here](https://github.com/gojek/turing/pull/198/files?short_path=4f18af2#r874426063)), should we just remove `isDisabled` since there no way the 'Add Rule' button will ever be disabled?

Reply:
Thanks for catching - at first, I didn't enforce the presence of at least one route from the routes panel and I had modified this condition (since all routes could be deleted). I introduced that check later but failed to update this. But thinking about it now, it makes sense to keep the old check here (have at least 2 routes to be able to create a rule) - so I put it back.